source/nodejs/adaptivecards/src/logging/Microsoft1DSLogger/Microsoft1DSLogger.ts

```diff
@@ -0,0 +1,87 @@
+import { ApplicationInsights } from '@ms/1ds-analytics-web-js'; //Imported for 1DS telemetry SDK
```

Comment:
Follow pattern for pulling in optional dependencies" [https://docs.npmjs.com/files/package.json#optionaldependencies](https://docs.npmjs.com/files/package.json#optionaldependencies)